stable/kubecost/charts/opencost/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: opencost
+description: A Helm chart for Kubernetes
+type: application
+version: 1.99.0
```

Comment:
My only qualm about starting the version at the appversion is that the upstream opencost chart might start at a lower version (like if they started at 0.1.0). Let's make sure that we can upgrade as seamlessly as possible to using the upstream chart once it is published!

Reply:
yeah i initially started at `0.1.0` but thought it might make it easier to align with the cost-analyzer chart. Im good with sticking to `0.1.0`